src/components/forms/cbr/CbrZoneExclusionAddressForm.js

```diff
@@ -0,0 +1,160 @@
+            onChange={this.handleInputChange}
+            invalidCallback={() =>
+              this.props.invalidCallback(this.state, this.props)
+            }
+            invalidText={this.props.invalidTextCallback(this.state, this.props)}
+            hideHelperText={true}
+          />
+          <IcseTextInput
+            id={this.props.data.name + "-cbr-account-id"}
+            componentName={this.props.data.name + "-cbr-zone"}
+            field={"account_id"}
+            value={this.state.account_id}
+            labelText={"Account ID"}
+            onChange={this.handleInputChange}
+            hideHelperText={true}
+            className="fieldWidthSmaller"
+            {...cbrInvalid("account_id", this.state.account_id)}
+          />
+          <IcseTextInput
+            id={this.props.data.name + "-cbr-zone-location"}
+            componentName={this.props.data.name + "cbr-zone-location"}
+            className={"fieldWidthSmaller"}
+            labelText={"Location"}
+            field="location"
+            value={this.state.location}
+            onChange={this.handleInputChange}
+            hideHelperText={true}
+            {...cbrInvalid("location", this.state.location)}
+          />
+        </IcseFormGroup>
+        <IcseFormGroup>
+          <IcseTextInput
+            id={this.props.data.name + "-cbr-zone-service-name"}
+            componentName={this.props.data.name + "cbr-zone-service-name"}
+            className={"fieldWidthSmaller"}
+            labelText={"Service Name"}
+            field="service_name"
+            value={this.state.service_name}
+            onChange={this.handleInputChange}
+            hideHelperText={true}
+            {...cbrInvalid("service_name", this.state.service_name)}
+          />
+          <IcseTextInput
+            id={this.props.data.name + "-cbr-zone-service-type"}
+            componentName={this.props.data.name + "cbr-zone-service-type"}
+            className={"fieldWidthSmaller"}
+            labelText={"Service Type"}
+            field="service_type"
+            value={this.state.service_type}
+            onChange={this.handleInputChange}
+            hideHelperText={true}
+            {...cbrInvalid("service_type", this.state.service_type)}
+          />
+        </IcseFormGroup>
+        <IcseFormGroup>
+          <IcseSelect
+            labelText="Type"
+            name="type"
+            formName={this.props.data.name + "cbr-zone-type"}
+            groups={["IP Address", "IP Range", "Subnet", "VPC", "Service Ref"]}
+            value={typeNameMap[this.state.type]}
+            handleInputChange={this.handleInputChange}
+            invalidText="Select a Type"
+            className="fieldWidthSmaller"
+          />
+          <IcseTextInput
+            id={this.props.data.name + "-cbr-zone-value"}
+            componentName={this.props.data.name + "cbr-zone-value"}
+            className={"fieldWidthSmaller"}
+            labelText={"Value"}
```

Comment:
should the placeholder for this input change depending on what type is specified? ie, IP Address => x.x.x.x